pulsar-functions/dist/pom.xml

```diff
@@ -0,0 +1,109 @@
+<!--
```

Comment:
Is this pom still needed or should we just rollover into parent dist module?

Reply:
yes. we should rollover into the parent dist module. probably better to do after merging this, to avoid touch files outside this functions module.